config/lib/app.js

```diff
@@ -4,24 +4,18 @@
  * Module dependencies.
  */
 var config = require('../config'),
-  mongoose = require('./mongoose'),
+  mongooseConfig = require('./mongoose-config'),
```

Comment:
Disambiguate between our mongoose file (`./mongoose`) and the mongoose library (`mongoose`).